packages/gatekeeper/src/helpers/services.ts

```diff
@@ -28,13 +28,13 @@ type HTTPMethod = 'GET' | 'POST' | 'get' | 'post';
  * @param {string} method - the HTTP method to use
  */
 export async function oauth2Callback(
```

Comment:
Please update the docstring of this function.

Reply:
addressed https://github.com/onaio/js-tools/pull/164/commits/6e2fd0e49af1596ed663bdd74c3e9c7f5f548ed6